src/main/java/com/sumologic/jenkins/jenkinssumologicplugin/sender/LogSender.java

```diff
@@ -112,6 +127,11 @@ private void createHeaders(final PostMethod post, final String sumoName,
             post.addRequestHeader("Content-Type", contentType);
```

Comment:
What happens if Content-Type is null?

Reply:
We have a condition to check valid content type. Graphite or carbon2. If it not valid, we do not set a content type.

Reply:
Content type was added for metric data only.